src/modules/master_transforms/src/master_tf.cpp

```diff
@@ -0,0 +1,101 @@
+#include "master_tf.h"
+
+/*
+NOTE: This node prepares the tf message to broadcast by using the input of the rotated yaw message
+      (according to REP-105) and performing the 2D rotation required from sensor location to front-axle. 
+      Only the x-position is offset (with respect to the vehicle on the red car) so there is no y-position component in the 
+      2-D transform. 
+
+INPUTS: TOPIC:  /localization/rotated_yaw
+                Msg: std_msgs::Float32
+        TOPIC:  /localization/calibrated_yaw
+                Msg: std_msgs::Float32
+
+OUTPUTS: Geodesy TF Broadcast
+         Lidar TF Broadcast
+*/
+
+namespace fusionad
+{
+namespace master_tf
+{
+namespace master_tf_node
+{
+    MasterTfNode::MasterTfNode()
+    {
+    }
+
+    MasterTfNode::~MasterTfNode()
+    {
+    }
+
+    void MasterTfNode::initRosComm()
+    {
+        // initializing the timer
+        master_tf_timer = masterTfNode_nh.createTimer(ros::Duration(0.02), &MasterTfNode::timerCallback, this);
+
+        // initializing the subscribers
+        rotated_yaw_sub = masterTfNode_nh.subscribe("/localization/rotated_yaw", 50, &MasterTfNode::rotatedYawCallback, this);
+        calibrated_yaw_sub = masterTfNode_nh.subscribe("/localization/calibrated_yaw", 10, &MasterTfNode::yawCallback, this);
+    }
+
+    void MasterTfNode::yawCallback(const std_msgs::Float32& cal_yaw_msg)
+    {
+        calibrated_yaw = cal_yaw_msg.data;
+        calibration_complete = true;
+    }
+
+    void MasterTfNode::rotatedYawCallback(const std_msgs::Float32& rot_yaw_msg)
+    {
+        rot_yaw = rot_yaw_msg.data;
+    }
+
+    void MasterTfNode::timerCallback(const ros::TimerEvent& event)
+    {
+        /*
+        GPS to front axle: -34.5" = -0.8763 [m]
```

Comment:
Can you mark these values as for the redcar in this comment somewhere, and make an issue to accept these parameters in some config or launch file?

Reply:
also maybe assign these as constants (on initialization) in a future issue. can probably include that in the issue to accept these values as params

Reply:
also maybe assign these as constants (on initialization) in a future issue. can probably include that in the issue to accept these values as params

Reply:
#158